cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -1213,6 +1214,26 @@ func checkDataAndSavePairing(rootdir string) (bool, error) {
 	return subtle.ConstantTimeCompare(marker1, marker2) == 1, nil
 }
 
+var (
+	// can be mocked in tests
+	partSrcPollIterations = 120
+	partSrcPollWait       = 500 * time.Millisecond
+)
```

Comment:
minor: I'd rather mock the whole function, because now the tests cannot verify that `waitPartSrc()` is called with the right `partSrc` parameter or inject an error.

Reply:
Yeah, thanks for this suggestion, this is great